examples/serverless/lambda-http-ddb-datasources/schema.graphql

```diff
@@ -0,0 +1,43 @@
+schema {
+	query: Query
+	mutation: Mutation
+}
+
+type Query {
+	listPosts: [Post]
+	listUsers: [User]
+	listTodos: [Todo]
+	getPost(id: ID!): Post
+	getUser(id: ID!): User
+	getTodo(id: ID!): Todo
+}
+
+type Mutation {
+	addTodo(input: AddTodoInput): Todo
+}
+
+# Lambda Datasource
```

Comment:
not a fan of putting references to the data source and their type in the schema since this is mainly typed to the resolver and can change at any time w/o impacting the schema.

Reply:
That makes sense, good point that it can change without impact. Removed comments